system/system_error_monitor/src/system_error_monitor_core.cpp

```diff
@@ -374,6 +375,11 @@ void AutowareErrorMonitor::onAutowareState(
 void AutowareErrorMonitor::onControlMode(
   const autoware_auto_vehicle_msgs::msg::ControlModeReport::ConstSharedPtr msg)
 {
+  // used for indicating that onControlMode callback is called at least once.
+  if (!control_mode_activated_) {
+    control_mode_activated_ = true;
+  }
+
```

Comment:
About this one, the new `control_mode_activated_` flag:
Reference from [the sheet](https://docs.google.com/spreadsheets/d/1z6RTyKQuaERNli5b-iCdj1ottrK84h_RUKbKmkc7YxU/edit?usp=sharing) Errors sheet Row 3:
  > This error message is intended to appear when `/vehicle/status/control_mode` is no longer subscribed and not sending data to Autoware  `system_error_monitor`.
  > The error message happens when an Autoware user starts planning simulator and the vehicle start location is not set you (simulator not initialized  yet). 
  > Once the vehicle start location is set and simulator is initialized, the topic publishes data and the error goes away.

In real vehicle, this error message could be expected to be seen even when the topic is never received.

I have 2 proposals to address these:

## 1. System Error Monitor Init Phase

Allow system_error_monitor node to have an initial countdown (user can set the seconds) and It won't report anything for this period. After it ends, it will publish as usual.

This period can be set to something like 10s for planning simulator maybe.

## 2. Autoware RViz Side Panel Visualization

We could have this somehow shown as red in the future side panel for this topic.
And once this error is gone, it would turn green for this message.

(This panel issue still hasn't been created so we can skip this solution for now.)

cc. @mitsudome-r 

@ahmeddesokyebrahim for now, let's revert this change and we can create another issue to resolve this with the `system_error_monitor` node.

Reply:
I have reverted the proposed solution and opened an [issue](https://github.com/autowarefoundation/autoware.universe/issues/6383).

Reply:
Thanks!